packages/arb-token-bridge-ui/src/util/wagmi/setup.ts

```diff
@@ -13,15 +13,16 @@ import { isTestingEnvironment } from '../CommonUtils'
 
 const chainList = isTestingEnvironment
   ? [
-      local,
-      arbitrumLocal,
       // mainnet, arb1, & arb nova are for network switch tests
       mainnet,
       arbitrum,
       arbitrumNova,
       // goerli & arb goerli are for tx history panel tests
       goerli,
-      arbitrumGoerli
+      arbitrumGoerli,
+      // add local environments during testing
+      local,
```

Comment:
From Fionna,
> wagmi by default tries to switch to the first allowed chain in the list
> we can move mainnet to the first item (maybe move local & arbitrumLocal to the bottom)

This fixes the issue of Connect with metamask trying to connect to localhost network locally, instead of one of our supported networks.